sintr_analysis_server_example/lib/query/versions.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library sintr_worker_lib.query.versions;
+
+import '../session_info.dart';
+import '../query.dart';
+
+class VersionMapper extends Mapper {
+  @override
+  map(String ln) {
+    Map sessionInfo = parseSessionInfo(null, ln);
+    var clientDate = new DateTime.fromMillisecondsSinceEpoch(
+        int.parse(sessionInfo[CLIENT_START_TIME]));
+    var dateString = "${clientDate.year}-${clientDate.month}-${clientDate.day}";
```

Comment:
I'm guessing that this will be a very common key. Should add this to the session info as CLIENT_START_DATE ?